markdown/org/docs/patterns/simon/instructions/en.md

```diff
@@ -85,7 +85,7 @@ You can now put your cuffs aside, we'll attach them later.
 
 ![Sew collar and undercollar together](3a.png)
 
-Place both collar parts together with the good sides, and sew them together at the standard seam allowance.
+Place both collar parts with the good sides together, and sew them together at the standard seam allowance.
```

Comment:
Can you use it like this:
```
with _good sides together_
```

That way it will get picked up by our jargon plugin

Reply:
Done :)